CHANGELOG.md

```diff
@@ -10,13 +10,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 
 ### Fixed
-- Bug where you could not use remote back button to exit YouTube if it was visited from Pocket (#1584)
-- Fixed 4K YT bug (#277)
-- Backing from the navigation overlay when on the first site in your backstack will now properly exit the app (#1916)
-- Fixed Youtube back bug (#1939)
-- Workaround for "grey screen" when returning to YouTube from Amazon homescreen (#1865)
+
+# [3.6-RO] 2019-03-26
+#*Released to Fire TV 4K, staged roll-out*
+### Fixed
 - Bug that prevented the toolbar back button from working on YouTube (#1927)
 
+## [3.5-A] - 2019-03-26
```

Comment:
There is no link for this release, so we should remove the square brackets.
Same thing for the 3.4-B release!